lib/menu.rb

```diff
@@ -0,0 +1,13 @@
+class Menu
+
+  attr_reader :list
+
+  def initialize
```

Comment:
What if we wanted to provide our own menu - for example we want this code to serve both chinese and pizza restaurants? How could we do this?